### DISCUSSION TITLE
Newest version (2.8.0) missing show(ISO)WeekNumbers property

### DISCUSSION THREAD

**Discussion by GamendeBrian** [malzariey/filament-daterangepicker-filter#116] [Q&A] [answered]
When reading the docs I saw the showWeekNumbers property and wanted to use it, but it said the property didn't exist while I was on the newest version.

Checked the main branch of your repo and saw that it existed there, but not on the newest release -> 2.8.0.

Could you publish a new release with this new functionality.
I'd really like to use it!

Thanks.

**Comment by malzariey** [comment, marked as answer]
Thank you, Published

**Reply by GamendeBrian** [reply]
Thanks!
